internal/cli/cloud_manager_clusters_apply.go

```diff
@@ -32,7 +32,7 @@ type cmClustersApplyOpts struct {
 	*globalOpts
 	filename string
 	fs       afero.Fs
-	store    store.AutomationStore
+	store    store.AutomationPatcher
```

Comment:
Store now has also a list of cluster which we don't need so making this interface more flexible